hooks/post_gen_project.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+import shutil
+
+
+def remove_frontend_files():
+    files_to_remove = [
+        ".eslintignore",
+        ".eslintrc.js",
+        ".nvmrc",
+        ".prettierignore",
+        ".stylelintignore",
+        ".stylelintrc.js",
+        "package.json",
+        "prettier.config.js",
+        "tsconfig.json",
+        "webpack.config.js",
+    ]
+    dirs_to_remove = [
+        Path("{{ cookiecutter.__project_name_snake }}", "static_src"),
+        Path("{{ cookiecutter.__project_name_snake }}", "static")
```

Comment:
question: should this be removed? What if I want to add good ole` CSS/JS?

Reply:
I don't have really strong views either way on this. Either position is defensible. Couple of thoughts:

- "good ole' CSS/JS" is included in most reasonable definitions of "frontend" :) Maybe `use_frontend` is the wrong name for this concept?
- If you want to include css/js, creating a couple of empty directories to put your files in isn't a big deal. A template like this gives you a starting point to extend from.

Reply:
`use_frontend_tooling`?
but fair point on manually creating a few empty dirs if you need them